### PR TITLE
Replace info list item with empty state component for searchbar example

### LINKED ISSUE #31: Searchbar example should use EmptyState component
#### Describe the desired behavior
When there are no results in the searchbar design pattern example an EmptyState component should be used to show this.

#### Describe the current behavior
When there are no results in the searchbar design pattern example an InfoListItem is being used to show no results.

#### Is this request related to a current issue?
Angular: See https://github.com/pxblue/angular-design-patterns/issues/61
ReactNative: See https://github.com/pxblue/react-native-design-patterns/issues/17

### PR DESCRIPTION
**Pull request by daileytj**
Fixes #31.

#### Changes proposed in this Pull Request:
- Replace info list item with empty state component for searchbar example when there are no results.

### CODE REVIEW

File: src/pages/app-bar/search-bar/index.tsx [hunk 1]
```diff
@@ -209,7 +209,19 @@ export const SearchBar = (): JSX.Element => {
...
+                            flexDirection: 'column',
+                            height: `calc(100vh - ${theme.spacing(8)}px)`,
+                        }}
+                    >
+                        <EmptyState
+                            icon={<Error fontSize={'inherit'} />}
+                            title={'0 results'}
+                            description={'No matching presidents'}
+                        />
+                    </div>
```
Comment (huayunh) [inline src/pages/app-bar/search-bar/index.tsx:224]: Can we take this logic outside the `<List>`, and add some inline comments to it?

**Review by jeffvg — APPROVED** [review]
looks good

File: src/pages/app-bar/search-bar/index.tsx (hunk 1 above)
Comment (daileytj) [reply src/pages/app-bar/search-bar/index.tsx:224]: What kind of inline comments do you suggest?

Comment (huayunh) [reply src/pages/app-bar/search-bar/index.tsx:224]: Now that I think twice about it... maybe a comment is not necessary :P

Comment (huayunh) [reply src/pages/app-bar/search-bar/index.tsx:224]: Looking good otherwise

**Review by huayunh — APPROVED** [review]

**Pull request merged**